### DISCUSSION TITLE
[2024/05/13] 데일리스크럼

### DISCUSSION THREAD

**Discussion by Galmaeki** [10aeat/10aeat_Backend#53] [데일리 스크럼]
### 데일리스크럼 전 PM, BE 논의 사항
- 프로필 이미지
  - 기본 이미지 사용
  - mvp 이후 프사 변경 기능 추가 논의 필요
- 회원 가입
  - 이메일 인증이 아닌 관리자 승인 으로 변경
  - 회원 가입시 동/호수 입력 필수
    - api명세서 수정 필요
    - 동/호수 여러개인 경우 마이페이지에서 추가 가능
  - 회원가입 완료와 동시에 로그인 처리
- 이슈
  - api명세서 작성된 대로 진행 예정
- 검색
  - 검색시 제목, 내용, 관리업체, 검사계획 항목 등에서 포함여부로 검색
  - 피그마에 검색 예시 페이지 참고
- 마이페이지 필요 api
  - 내가 단 댓글 모아보기
  - 내가 저장한 글 모아보기
  - 알림 히스토리
  - 동 호수 추가

#### BE 안건
- api명세서 fe에 전달
  - PM팀에 관리자 사이드 전달
  - 재현)알림 아직 미구현이지만 롱폴링이 긍정적일것 같다

#### UI팀 질문
- 오피스너 앱에 탑재로 알고 있는데 어떤식으로 갈지 문의
  - 개발팀도 어떻게 진행될지 알 수 없어 문의 남길 예정
  - 오피스너가 우리한테 넘어오는 링크만 만들어주는건지, 아니면 우리가 오피스너 자원 공유 가능한지

- 추가 기능 언급
  - 월별 요약 보기에 이번달 안건 몇개인지 홈화면에서도 미리 볼 수 있게
  - be : 구현은 어렵지 않을것 같음
  - fe : 홈 화면이 아직 안나와있기 때문에 미정

**Comment by asd42270** [comment]
감사합니다!!! 숙지해서 가겠습니다...하하하
